keyboards/the_royal/sukari/sukari.h

```diff
@@ -0,0 +1,35 @@
+#pragma once
+
+#include "quantum.h"
+
+#define XXXX KC_NO
+
+#define LAYOUT_true( \
+    k000, k001, k002, k003, k004, k005, k006, k007, k008, k009, k010, k011, \
+    k100, k101, k102, k103, k104, k105, k106, k107, k108, k109, k110, k111, \
+    k200, k201, k202, k203, k204, k205, k206, k207, k208, k209, k210, k211, \
+    k300, k301, k302, k303, k304, k305, k306, k307, k308, k309, k310, k311, \
+    k400,    k401,                   k406,                   k410,    k411  \
+) \
+{ \
+    {k000, k001, k002, k003, k004, k005, k006, k007, k008, k009, k010, k011}, \
+    {k100, k101, k102, k103, k104, k105, k106, k107, k108, k109, k110, k111}, \
+    {k200, k201, k202, k203, k204, k205, k206, k207, k208, k209, k210, k211}, \
+    {k300, k301, k302, k303, k304, k305, k306, k307, k308, k309, k310, k311}, \
+    {k400, k401, XXXX, XXXX, XXXX, XXXX, k406, XXXX, XXXX, XXXX, k410, k411}  \
+}
+
+#define LAYOUT_full( \
```

Comment:
I believe that the "proper" notation that we recommend for this is `LAYOUT_all`
```suggestion
#define LAYOUT_all( \
```

Reply:
Understood.  I'll keep that in mind in the future.